core/views.py

```diff
@@ -188,7 +188,7 @@ class TeamView(generics.RetrieveUpdateDestroyAPIView):
     """
 
     def get_permissions(self):
-        if self.request.method == "GET":
+        if self.request.method in ["GET"]:
```

Comment:
Instead of simply using `GET`, using `SAFE_METHODS` would be a better choice.

Reply:
That change was  commited by mistake. For now let it be explicitly clear about `GET` method only.